en/docs/reference/mediators/respond-mediator.md

```diff
@@ -32,39 +32,48 @@ the rest of the CRF case configuration is left intact, allowing you to
 revert to the original behavior in the future by removing the Respond
 mediator if required.
 
-```xml
-<proxy name="SimpleProxy" transports="http https" startonload="true" trace="disable" xmlns="http://ws.apache.org/ns/synapse">
-    <target>
-         <inSequence>
-      <switch source="//m0:getQuote/m0:request/m0:symbol" xmlns:m0="http://services.samples">
-        <case regex="WSO2">
-          <property name="symbol" value="Great stock - WSO2"/>
-          <send>
-            <endpoint>
-              <address uri="http://localhost:9000/services/SimpleStockQuoteService"/>
-            </endpoint>
-          </send>
-        </case>
-        <case regex="CRF">
-
-          <respond/>
+=== "Proxy Service"
+    ```xml
```

Comment:
Check whether syntax formatting appears correctly. Because of the tab, it seems syntax formatting is not appearing correctly.

Reply:
updated the formatting.